packages/compass-e2e-tests/tests/logging.test.js

```diff
@@ -198,25 +198,26 @@ describe('Logging and Telemetry integration', function () {
           },
         },
         {
-          s: 'I',
-          c: 'COMPASS-CONNECT',
-          id: 1_001_000_010,
-          ctx: 'Connect',
-          msg: 'Resolved SRV record',
+          s: 'E',
+          c: 'DEVTOOLS-CONNECT',
+          id: 1_000_000_041,
+          ctx: 'compass-deps',
+          msg: 'Missing optional dependency',
           attr: (actual) => {
-            expect(actual.from).to.match(/^mongodb:\/\/localhost:27018/);
-            expect(actual.to).to.match(/^mongodb:\/\/localhost:27018/);
+            // https://jira.mongodb.org/browse/COMPASS-4768
+            // This requires some dark webpack magic to work, apparently.
+            expect(actual.name).to.equal('os-dns-native');
```

Comment:
@gribnoysup The specific issue here is the way that the `bindings` module tries to get the path of the file that calls it in order to locate the addon (it fails because the filenames on the stack are reported as `undefined`). I tried adding `os-dns-native` everywhere where the other native addons are also mentioned in the source code, but that didn’t resolve that issue. Maybe you happen to know what’s up with that?